pages/common/lynx.md

```diff
@@ -0,0 +1,23 @@
+# lynx
+
+> A Command-line web browser.
```

Comment:
Perhaps this will read better?
```suggestion
> Command-line web browser.
```

Also, is there a "more information" link we can add here too - for example to a homepage perhaps?